src/content/docs/products/splashkit/documentation/splashkit-online/research-and-findings/APISupportTests.md

```diff
@@ -0,0 +1,75 @@
+| Networking       | No Support        | No Support        | All networking functionality is replaced with stubs currently, due to no cURL support.                                                                                                                                                                                   |
+| Physics          | Full Support      | Full Support      |                                                                                                                                                                                                                                                                          |
+| Raspberry        | No Support        | No Support        | Disabled during build.                                                                                                                                                                                                                                                   |
+| Resource Bundles | Full Support      | Full Support      |                                                                                                                                                                                                                                                                          |
+| Resources        | Full Support      | Full Support      |                                                                                                                                                                                                                                                                          |
+| Sprites          | Full Support      | Full Support      |                                                                                                                                                                                                                                                                          |
+| Terminal         | Partial Support   | Limited Support   | **JavaScript**: it all _works_, but no terminal input bar. 'Input' _popup_ appears, that has confusing behaviour of only sending data once _cancelled_. This may be fixed very shortly however. <br/>**C++**: all the _read_ functions return immediately with no input. |
+| Timers           | Full Support      | Full Support      |                                                                                                                                                                                                                                                                          |
+| Utilities        | Near Full Support | Near Full Support | [Display Dialog](http://localhost:4321/api/utilities/#display-dialog) does not work, as it enters a busy loop that freezes the page.                                                                                                                                     |
```

Comment:
I believe you want this link here and not localhost: https://splashkit.io/api/utilities/#display-dialog